src/firebase_functions/private/manifest.py

```diff
@@ -87,7 +87,7 @@ class BlockingTrigger(_typing.TypedDict):
 
 class VpcSettings(_typing.TypedDict):
     connector: _typing_extensions.Required[str]
-    egressSettings: _typing_extensions.NotRequired[str]
+    egressSettings: _typing_extensions.NotRequired[str | _util.Sentinel]
```

Comment:
do you know why we don't need to support similar type fix (i.e. adding _util.Sentinel) to every other option that's resettable?

Reply:
Other options that were resettable already had this from the original DEFAULT_VALUE - this one just got missed originally